client/gutenberg/extensions/classic-block/edit.jsx

```diff
@@ -23,22 +23,56 @@ export class ClassicEdit extends Component {
 	}
 
 	storeEditor = ref => {
+		//By the time editor lifecycle events like PostProcess fire, this editor instance is already marked for removal
+		//so any content manipulation doesn't get reflected in this block.
+		//
+		//For now, when we detect an unmount, get content from the editor marked to be destroyed and update block attributes.
+		//https://github.com/Automattic/wp-calypso/issues/29643
```

Comment:
small nitpick, but I'm used to seeing a space after `//` and before the text. are you comfortable adding one in on these lines?

other than that I like to use these comments as examples to others of how we can help other developers or our future selves by explaining what isn't obvious from the code itself - bravo

Reply:
Sure, I can update! I'm hoping this patch is a temporary fix until we get a better handle on the intended lifecycle loops.

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/29858/commits/108abb15de335f54ec4c9f45edfb6f52416e3329